plugins/copr.py

```diff
@@ -479,6 +480,38 @@ def _download_repo(self, project_name, repo_filename, chroot=None):
         shutil.copy2(f.name, repo_filename)
         os.chmod(repo_filename, stat.S_IRUSR | stat.S_IWUSR | stat.S_IRGRP | stat.S_IROTH)
 
+    def _runtime_deps_warning(self, copr_username, copr_projectname, chroot=None):
+        self.base.reset(repos=True)
+        self.base.read_all_repos()
+
+        repo = self._get_copr_repo(self._sanitize_username(copr_username), copr_projectname)
+
+        runtime_deps = []
+        for repo_id in repo.cfg.sections():
+            if repo_id.startswith("copr:"):
+                continue
+            runtime_deps.append(repo_id)
+
+        if not runtime_deps:
+            return
+
+        print(_("The Copr project {0} uses the following repositories as runtime dependencies:"
+              .format(copr_projectname)))
+        dep_idx = 1
+        for repo_id in runtime_deps:
+            print("{0}. [{1}]".format(dep_idx, repo_id))
```

Comment:
Can you do this?  It makes the output more airy.  If you disagree, no problem (it's matter of taste I guess).
```
--- /usr/lib/python3.8/site-packages/dnf-plugins/copr.py        2020-04-21 07:39:33.965821445 +0200
+++ /tmp/backup.py      2020-04-21 07:38:04.733792653 +0200
@@ -495,15 +495,15 @@
         if not runtime_deps:
             return
 
-        print(_("\nThe Copr project {0} uses the following repositories as runtime dependencies:"
+        print(_("The Copr project {0} uses the following repositories as runtime dependencies:"
               .format(copr_projectname)))
         dep_idx = 1
         for repo_id in runtime_deps:
-            print("\n{0}. [{1}]".format(dep_idx, repo_id))
+            print("{0}. [{1}]".format(dep_idx, repo_id))
             print("   baseurl={0}".format(repo.cfg.getValue(repo_id, "baseurl")))
             dep_idx += 1
 
-        msg = _("\nDo you want to keep them enabled?")
+        msg = _("Do you want to keep them enabled?")
         if self.base._promptWanted():
             if self.base.conf.assumeno or not self.base.output.userconfirm(
                     msg='{} [y/N]: '.format(msg), defaultyes_msg='{} [Y/n]: '.format(msg)):
```